test/pod.persona.test.ts

```diff
@@ -0,0 +1,229 @@
+import { ethers } from 'ethers';
+import axios from 'axios';
+import { init, getPod } from '../src';
+import * as createSafe from '../src/lib/services/create-safe-transaction';
+import { userAddress2, constructGqlGetUsers, getSafeTransactionFixture } from './fixtures';
+import * as fetchers from '../src/fetchers';
+import * as utils from '../src/lib/utils';
+
+// Some of these tests run long.
+jest.setTimeout(7500);
+
+const expectedPod = {
+  admin: '0xf0C7d25c942264D6F21871c05d3dB3b98344b499',
+  id: 6,
+  name: '1-member-pods.pod.eth',
+  safe: '0xC99cE75a83B068DABF48c48f91C62a30fe0afBd2',
+  member: '0x61De0bbb6C8215Af3f821FE4884A28bc737f98D3', // random member
+};
```

Comment:
🤔 Do you mind setting variables for all of these addresses? And maybe for different personal types?  Ex:
`const adminAddress = '0xf0C7d25c942264D6F21871c05d3dB3b98344b499'`

I think it'll make the tests easier to follow.

Reply:
Of course! Don't mind at all. Will update now.